lib/services/circleci.rb

```diff
@@ -9,7 +9,11 @@ class Service::Circleci < Service
 
   default_events Service::ALL_EVENTS
 
+  string :domain
+  white_list :domain
+
   def receive_event
+    http.ssl[:verify] = false
```

Comment:
Is there any way to require customers to have SSL setup? With this verify setting, it would be very simple to send the payload to an insecure server.